### PR TITLE
Add comparison to other ad networks

### PR DESCRIPTION
**Pull request by ericholscher**
The goal here is to show how we're different..
I wasn't sure how direct we wanted to be,
so I made the link and some copy focus mostly on Carbon.
I did include Stack Overflow as well,
since I think they are our other major competitor here.

I considered having both `Comparison to Carbon/Stack Overflow` in the footer linking to the same page,
but wasn't sure the best approach?

### CODE REVIEW

**Comment by ericholscher** [comment]
Updated to only focus on Carbon

File: content/pages/vs-carbon-ads.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+This page talks a bit more about how we're different,
+and why you might want to choose us instead of Carbon Ads.
+
+[TOC]
+
+## **Privacy**: We never set a cookie on your site, or load third-party scripts.
+
+When your user goes to your website, they are assuming they will only be sending data to you. Some users consider it a breach of trust if you were to share that data with other third parties, and it might be a breach of a privacy law to do this without user consent.
+
+These networks will often set cookies on your browser session, as well as load scripts hosted by Google into a publishers web page. You can see a screenshot of Carbon Ads loading a Google script here:
```
Comment (davidfischer) [inline content/pages/vs-carbon-ads.md:16]:
```suggestion
These networks will often set cookies on your browser session, as well as load scripts hosted by Google into a publishers web page. You can see a screenshot of Carbon Ads loading a Google tracking pixel here:
```

Shots fired! Still, let's be precise.

File: content/pages/vs-carbon-ads.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+Title: What Makes EthicalAds a Great Carbon Ads Alternative
```
Comment (davidfischer) [inline content/pages/vs-carbon-ads.md:1]:
```suggestion
Title: What Makes EthicalAds a Great Carbon Ads Alternative for Your Site
```

I don't feel that strongly but this change might make it more obvious that this is geared toward publishers.

File: content/pages/vs-carbon-ads.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+## **Privacy**: We never set a cookie on your site, or load third-party scripts.
+
+When your user goes to your website, they are assuming they will only be sending data to you. Some users consider it a breach of trust if you were to share that data with other third parties, and it might be a breach of a privacy law to do this without user consent.
+
+These networks will often set cookies on your browser session, as well as load scripts hosted by Google into a publishers web page. You can see a screenshot of Carbon Ads loading a Google script here:
+
+<div class="m-5 postimage">
+  <a href="{static}../images/posts/2021-third-party-resources.png">
+  <img class="w-50 rounded mx-auto d-block"  src="{static}../images/posts/2021-third-party-resources.png" alt="An ad network loading third-party resources">
+  </a>
```
Comment (davidfischer) [inline content/pages/vs-carbon-ads.md:21]:
```suggestion
  </a>
  <p><small>Carbon sometimes loads third-party resources from Google or other trackers</small></p>
```

**Review by davidfischer — APPROVED** [review]

**Pull request merged**